### DISCUSSION TITLE
How can I install Copilot on my laptop (on VisualStudio)?

### DISCUSSION THREAD

**Discussion by DavidThielen** [community/community#49199] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi all;

I am trying to install Copilot on VisualStudio on my laptop. It installed on my desktop a couple of weeks ago with no problem. But when I try to download it in Visual Studio 2022 I get:

![GitHubInstall](https://user-images.githubusercontent.com/52540224/222933441-a70bcd6a-6635-4a00-b8d0-1afc850cb802.png)

**Comment by iamcjwilliams** [comment]
@DavidThielen Are you still experiencing this issue?

**Comment by DavidThielen** [comment]
Still having the same problem. Can’t install because of it.


David Thielen- (ex)CEO/founder Windward Studios
Substack: Liberal & Loving It<https://liberalandlovingit.substack.com/>
LinkedIn<https://www.linkedin.com/in/davethielen/> | calendy<https://calendly.com/davethielen>
(720) 352-0676 – cell

**Comment by DavidThielen** [comment]
I rebooted one more time – and now it installs.

You can close this ticket.

Thanks – dave



David Thielen- (ex)CEO/founder Windward Studios
Substack: Liberal & Loving It<https://liberalandlovingit.substack.com/>
LinkedIn<https://www.linkedin.com/in/davethielen/> | calendy<https://calendly.com/davethielen>
(720) 352-0676 – cell

From: David Thielen
Sent: Monday, March 6, 2023 5:41 PM
To: community/community ***@***.***>; community/community ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [community/community] How can I install Copilot on my laptop (on VisualStudio)? (Discussion #49199)

Still having the same problem. Can’t install because of it.


David Thielen- (ex)CEO/founder Windward Studios
Substack: Liberal & Loving It<https://liberalandlovingit.substack.com/>
LinkedIn<https://www.linkedin.com/in/davethielen/> | calendy<https://calendly.com/davethielen>
(720) 352-0676 – cell
